osu.Game/Overlays/Changelog/ChangelogSingleBuild.cs

```diff
@@ -59,13 +61,13 @@ private void load(CancellationToken? cancellation, IAPIProvider api, OverlayColo
                 Thread.Sleep(10);
             }
 
-            if (build != null)
```

Comment:
This is an invalid if, 

and the incomplete `build` being passed to `ChangelogBuild`